src/metabase/api/pulse.clj

```diff
@@ -291,6 +291,8 @@
    :body    (preview/style-tag-from-inline-styles
              (html5
                [:head
+                [:meta {:nonce "%NONCE%" ;; this will be str/replaced by 'style-tag-nonce-middleware
+                        :charset "utf-8"}]
```

Comment:
This is nearly correct! You don't actually need that :nonce in this case. The nonce is needed for styles to pass our CSP, but setting the meta here doesn't need anything like that.

```suggestion
                [:meta {:charset "utf-8"}]
```